admin/tabs/activity/tracking/ActivityTrackingModel.ts

```diff
@@ -191,30 +175,40 @@ export class ActivityTrackingModel extends HoistModel {
         this.addReaction({
             track: () => {
                 const vals = this.formModel.values;
-                return [vals.startDay, vals.endDay, vals.maxRows];
+                return [vals.startDay, vals.endDay, vals.maxRows, this.filterChooserModel.value];
             },
             run: () => this.loadAsync(),
             debounce: 100
         });
 
         this.addReaction({
-            track: () => [this.cube.records, this.filterChooserModel.value, this.dimensions],
+            track: () => [this.cube.records, this.dimensions],
             run: () => this.loadGridAsync(),
             debounce: 100
         });
     }
 
     override async doLoadAsync(loadSpec: LoadSpec) {
-        const {enabled, cube, formModel} = this;
+        const {enabled, cube, filterChooserModel, formModel} = this;
         if (!enabled) return;
 
         try {
-            const data = await XH.fetchJson({
+            const data = await XH.fetchService.postJson({
                 url: 'trackLogAdmin',
-                params: formModel.getData(),
+                body: {
+                    ...formModel.getData(),
+                    filters: filterChooserModel.value
+                },
                 loadSpec
             });
 
+            const lookups = await XH.fetchJson({url: 'trackLogAdmin/lookups'});
+
+            filterChooserModel.fieldSpecs.forEach(spec => {
+                const {field} = spec;
+                if (lookups[field]) spec.values = lookups[field];
+            });
```

Comment:
We could run this lookups query just once, on init, especially if we had any reason to think it could be resource-intensive on the server. 

Reloading each time does have the advantage of picking up new discrete values that were added to the overall log since you first opened the tab. That's not something that's likely to happen _all_ that often, but of course it could. So I'm a bit on the fence.  I'd want to see how that call performs with a very large dataset, and we should generally be confident it performs well (e.g. those fields are marked for indexing)

If we keep reloading it each time, I'd suggest we kick that off in its own method and don't bother awaiting it here (or  letting it throw the main return for that matter, if it e.g. timed out).  While we don't want to plan for that call to fail, it is a bit ancillary. Certainly nice to not make the user wait on it before seeing the main results.

Reply:
Going to opt to move this logic into initialization for now